lib/plug/cowboy/stream.ex

```diff
@@ -25,6 +25,12 @@ defmodule Plug.Cowboy.Stream do
   end
 
   def early_error(_stream_id, reason, partial_req, resp, _opts) do
+    :telemetry.execute(
+      [:plug_cowboy, :early_error],
+      %{system_time: System.system_time()},
+      %{reason: reason, partial_req: partial_req, resp: resp}
```

Comment:
I am not really a fan of exposing Cowboy internals here. Maybe we should only expose the information we use in the error message below? Thoughts?

Reply:
I think as long as we get the HTTP status code & basic request info, most possible use cases will be covered, I'll make some tweaks...